README.md

```diff
@@ -2,178 +2,230 @@
 
 [![CircleCI](https://circleci.com/gh/openfun/arnold.svg?style=svg)](https://circleci.com/gh/openfun/arnold)
 
-Arnold is a tool to deploy dockerized applications to
-[OpenShift](https://www.openshift.com/) with
-[Ansible](https://www.ansible.com/). It was built by France Université Numérique
-to ease its infrastructure deployment.
-
-The current work mainly focuses on the [Open edX MOOC
-platform](https://open.edx.org/), but it can be considered as a generic tool to
-deploy dockerized applications.
+Arnold is a generic tool to deploy dockerized applications to
+[Kubernetes](https://kubernetes.io/) with [Ansible](https://www.ansible.com). It was built
+by France Université Numérique to ease its infrastructure deployment.
 
 ## Overview
 
-Arnold has been designed as a suite of Ansible playbooks and OpenShift object
-definition templates (Jinja2). We take advantage of the `openshift_raw` Ansible
-module to make Ansible talk with OpenShift.
-
-As a DevOps using this project, you will need to adapt OpenShift object
-templates to suite your needs or constraints and run playbooks to push your
-changes to your OpenShift instance that orchestrates your services.
+Arnold has been designed as a suite of Ansible playbooks and Kubernetes object
+definition templates (Jinja2). We take advantage of the `k8s` Ansible modules
+to make Ansible talk with Kubernetes.
 
 ## Requirements
 
 - [Docker](https://docs.docker.com/engine/installation/): we use docker to
   develop and run Arnold. This is a strict requirement to use this project.
-- [OpenShift's client](https://docs.openshift.org/latest/welcome/index.html)
-  (_aka_ `oc`): this CLI is used to communicate with the running OpenShift
-  instance you will use. Plus, it offers the possibility to run a minimal
-  cluster for development purpose (using a set of docker containers). You
-  can use the [`3.11` release of
-  `oc`](https://github.com/openshift/origin/releases/tag/v3.11.0). Please
-  avoid the `3.10` release because we encountered DNS issues with it.
-
-> Even if we recommend to only use `oc` and docker to work locally with Arnold,
-> you might also consider using
-> [MiniShift](https://docs.openshift.org/latest/minishift/getting-started/) as a
-> relevant alternative to run an isolated OpenShift cluster within a VM (please
-> refer to our [instructions to install
-> MiniShift](./docs/installation/minishift.md)).
-
-## Quick start
-
-**Disclaimer**: this quick start guide has been written with development /
-testing purpose in mind. If you are looking for more insights on how to use it
-in production, please refer to our [documentation](./docs/index.md).
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/):
+  This CLI is used to communicate with the running Kubernetes instance you
+  will use.
+- [k3d](https://k3d.io/): This tool is used to setup and run a lightweight
```

Comment:
this is probably optional no? I think the most common case is someone who already has a cluster with a cloud provider and wants to deploy to it so we should not bother them with the complexity of running k8s locally?

If you agree, it means that the bootstrap instructions below should instruct to connect to one's existing k8s cluster or propose to start a k3d as an option (with a link to another doc for that?)

Reply:
I agree with these points:
- `k3d` is an optional dependency
- we need instructions on how to connect to an existing Kubernetes cluster

Since this last part might be complicated (due to the service account requirement), I would be tempted to stick to using k3d in the bootstrap instructions since it's easier like this. But maybe i'm wrong and the real use case is to directly connect to a real kubernetes cluster. Your opinion will be appreciated after reading this.

Reply:
ok let's keep it like this for the moment!

Reply:
So for the moment I moved k3d to optional dependencies with a mention: https://github.com/openfun/arnold/pull/672/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R25-R28

Reply:
Then I removed all outdated documentation files. @madmatah are you ok with that?

Reply:
@sampaccoud Thanks! yes I'm ok with that